### DISCUSSION TITLE
Is wails the Go version of tauri? Where is its advantage? Hope to see the comparison between them

### DISCUSSION THREAD

**Discussion by hcg1023** [wailsapp/wails#1597] [Q&A] [answered]
Is wails the Go version of tauri? Where is its advantage? Hope to see the comparison between them

**Comment by leaanthony** [comment, marked as answer]
Wails was released before Tauri so is it the Go version of Tauri? 🤔 Wails is for Go developers, Tauri is for Rust developers. I can't give you much more of a comparison than that. I guess the websites are your best source of information.

**Comment by kishaningithub** [comment]
@hcg1023 For detailed comparison of wails vs tauri you can refer these links
- https://github.com/Elanis/web-to-desktop-framework-comparison
- https://github.com/tauri-apps/tauri/discussions/3521
